docs/gitbook/usage/envoy-progressive-delivery.md

```diff
@@ -0,0 +1,300 @@
+# Envoy Canary Deployments
+
+This guide shows you how to use Envoy and Flagger to automate canary deployments.
+
+### Prerequisites
+
+Flagger requires a Kubernetes cluster **v1.11** or newer and Envoy paired with [Crossover](https://github.com/mumoshu/crossover) sidecar.
+
+Create a test namespace:
+
+```bash
+kubectl create ns test
+```
+
+Install Envoy along with the sidecar with Helm:
+
+```bash
+git clone https://github.com/mumoshu/crossover.git
+
+cd crossover
+
+helm upgrade --install envoy stable/envoy \
+  --namespace test \
+  -f example/values.yaml \
+  -f example/values.services.yaml \
+  --set services.podinfo.smi.enabled=true
```

Comment:
Why do we need to specify the app service during install? Can this be added to Envoy afterwards?